doc/manual.rst

```diff
@@ -249,8 +249,11 @@ Identifiers & Keywords
 ----------------------
 
 Identifiers in Nim can be any string of letters, digits
-and underscores, beginning with a letter. Two immediate following
-underscores ``__`` are not allowed::
+and underscores, with the following restrictions:
+
+* begins with a letter
+* does not end with an underscore ``_``
+* two immediate following underscores ``__`` are not allowed::
```

Comment:
Capitalize the lines, they should start with capital letter IMHO.

Reply:
Items in a list can be lowercase. Don't nitpick.